### DISCUSSION TITLE
Migration from XMLHttpRequests to Fetch

### DISCUSSION THREAD

**Discussion by junaid-enp** [axios/axios#5840] [Q&A]
Currently Axios uses `XMLHttpRequests` under the hood. Is there any chances that Axios will be migrated to `Fetch`

**Comment by Quent1L** [comment]
It would be very interesting if axios migrated to `fetch` to minimise the size of this dependency as much as possible. The [redxios](https://www.npmjs.com/package/redaxios) library has partially taken over the axios API using `fetch`. Are there any plans for axios to migrate one day?

**Comment by landsman** [comment]
Check this: https://github.com/axios/axios/pull/5146

**Comment by jasonsaayman** [comment]
Fetch is now supported. You can check the docs
